src/Lykke.Job.BlockchainOperationsExecutor.Core/Domain/TransactionExecutions/TransactionExecutionAggregate.cs

```diff
@@ -197,9 +210,16 @@ public void OnSigned(string signedTransaction)
 
         public void OnBroadcasted()
         {
-            State = TransactionExecutionState.Broadcasted;
+            if (SourceAndTargetAddressesLockingMoment.HasValue)
```

Comment:
Let's introduce calculated property with clear name, to determine if exclusive lock workflow should be applied and use it everywhere instead of SourceAndTargetAddressesLockingMoment.HasValue